src/Servers/IIS/ResolveIisReferences.targets

```diff
@@ -71,6 +71,12 @@ with the right MSBuild incantations to get output copied to the right place.
     </When>
   </Choose>
 
+  <Target Name="_WarnAboutUnbuiltNativeDependencies"
+          BeforeTargets="Build"
+          Condition=" @(NativeProjectReference->Count()) != 0 AND '$(BuildNative)' == 'false' ">
+    <Warning Text="This project has native dependencies which were not built. Without this, tests may not function correctly. Run `build.cmd -native` to build native projects. Run `build.cmd -managed -native` to build both C# and C++." />
```

Comment:
cc @pakrym @jkotalik

Reply:
Does `build.cmd` without any flags still build both native and managed?

Reply:
yes